docs/tutorials/basics.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "7FV_m2u2Hny8"
```

Comment:
<p>this says it uses --pre, but it doesn't use it below in the install?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>